lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -80,9 +80,11 @@ class SafariPointerEventWorkaround {
 }
 
 class PointerBinding {
-  PointerBinding(this.flutterViewElement, this._keyboardConverter)
-    : _pointerDataConverter = PointerDataConverter(),
-      _detector = const PointerSupportDetector() {
+  PointerBinding(
+    this.flutterViewElement,
+    this._keyboardConverter, [
+    this._detector = const PointerSupportDetector(),
```

Comment:
I don't think we need this `PointerSupportDetector` anymore?

Reply:
This was (and still is) useful for testing